docs/targets/http.md

```diff
@@ -1,49 +1,53 @@
 # Event Target for HTTP
 
-This event target receives [CloudEvents][ce] and turns them into HTTP requests that consume external services.
+This event Target receives [CloudEvents][ce] and turns them into HTTP requests that consume external services.
 
-## Prerequisites
+## Prerequisite(s)
 
-The HTTP event target sends requests to arbitrary URLs and wraps responses in CloudEvents back to the caller. HTTP endpoints that are unauthenticated, use basic authentication or use custom header values for authentication, can be integrated using this target.
+The HTTP event Target sends requests to arbitrary URLs and wraps responses in CloudEvents back to the caller. HTTP endpoints that are unauthenticated, use basic authentication or use custom header values for authentication, can be integrated using this target.
```

Comment:
I agree, I was struggling a little to figure out what prerequisites to pull from these paragraphs. Does this look okay?
```
## Prerequisite(s)

- HTTP endpoints
- Password

## HTTP Endpoints

The HTTP event Target sends requests to arbitrary URLs and wraps responses in CloudEvents back to the caller. HTTP endpoints that are unauthenticated, use basic authentication or use custom header values for authentication, can be integrated using this target.

Responses from external HTTP endpoints are converted into CloudEvents and sent as a reply to the TriggerMesh Broker/Channel. It is important that the HTTP Target filters received events and cares about response event type and event source to avoid loops where those responses might end up being processed by the HTTP Target.

Requests from this HTTP Target will verify TLS certificates from the remote server if present. If the CA certificate at the server is self-signed, the public certificate needs to be added to the configuration, or alternatively mark the `Skip Verify` option.

## Password

If the remote endpoint requires basic authentication, the password needs to be created as a secret. Consult the [Secrets](../guides/secrets.md) guide for more information about
how to add the password as a secret.
```

Reply:
Sounds legit to me.